extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -267,11 +264,7 @@ export class VisibilityManager {
    * @return {!../../../src/layout-rect.LayoutRectDef}
    */
   getElementLayoutBoxTemp(element) {
-    const resource = this.resources_.getResourceForElementOptional(element);
```

Comment:
Do you know what this TODO is about? https://github.com/ampproject/amphtml/issues/29618
The behavior should be slightly worse for cases with a Resource right?

Reply:
Not sure what #29618 is about, tbh. This PR might fix it possibly. I'm going to ping it for re-eval once this PR is merged.

Reply:
It's actually have been removed.